code/src/App.js

```diff
@@ -1,9 +1,79 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
+import { API_HEART, API_LIST } from './components/Api'
+import ThoughtList from 'components/ThoughtList'
+import ThoughtForm from 'components/ThoughtForm'
+
+
 
 export const App = () => {
+
+    const [thoughtList, setThoughtList] = useState([])
+    // const [loading, setLoading] = useState(false) - never used due loading on likes
+    const [newThought, setNewThought] = useState ('')
+
+    useEffect(() => {
+      fetchThoughts();
+    }, []);
+  
+    //fetch 20 latest thoughts
```

Comment:
Your code is clear and easy to read so you could remove the comments explaining your sections.